BaseStation/Controller Controls/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTK.Input;
+using Scarlet.Communications;
+using Scarlet.Utilities;
+
+// this code will be used for the basestation and takes in all of the controller inputs then sends
+// them to the rover side. Whichever is client or server side has yet to be determined
+
+namespace ControllerServer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int PortTCP = 5287;
+            int PortUDP = 5288;
+            byte PacketID = 0; // Packet ID must be formalized, 0 is just a placeholder
+            Parse.SetParseHandler(0, PrintPacketData);
+
+            Log.SetGlobalOutputLevel(Log.Severity.ERROR);
+            Server.Start(PortTCP, PortUDP);
+            while (true)
+            {
+                if (Server.GetClients().Contains("Controller Parser"))
+                {
+                    GamePadState state = GamePad.GetState(0);
+                    Packet info = new Packet(new Message(PacketID,
```

Comment:
Can this be extracted into a method?

Reply:
Do you mean the while loop? if so I'll look into that

Reply:
I was thinking just in a separate method so it wasn't living in `Main`